guides/v2.2/frontend-dev-guide/layouts/layout-overview.md

```diff
@@ -99,27 +97,38 @@ The Magento application processes layout files in the following order:
+{: .bs-callout .bs-callout-info #info}
 
 ## Layout files validation
 
-After layouts are merged, Magento validates them. 
+After layouts are merged, Magento validates them.
 
-Layout validations and error handling depends on the [application mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html) in which you Magento instance runs:
+Layout validations and error handling depends on the [application mode] in which you Magento instance runs:
 
 - developer mode: syntax is validated in `.xml` and `.xsd` files, and `.xml` files are validated according to the xsd schema. If any validation fails, the hard failure with process halt occurs.
 
-- production or default modes: syntax is validated in `.xml` and `.xsd` files. If validation fails, errors are logged to the `var/log` directory without throwing {% glossarytooltip 53da11f1-d0b8-4a7e-b078-1e099462b409 %}exception{% endglossarytooltip %}. The validation according to the xsd schema is not performed. 
+- production or default modes: syntax is validated in `.xml` and `.xsd` files. If validation fails, errors are logged to the `var/log` directory without throwing {% glossarytooltip 53da11f1-d0b8-4a7e-b078-1e099462b409 %}exception{% endglossarytooltip %}. The validation according to the xsd schema is not performed.
 
 ## Related topics
 
-*	<a href="{{ page.baseurl }}/frontend-dev-guide/layouts/xml-instructions.html" target="_blank">Layout instructions</a>
-*	<a href="{{ page.baseurl }}/frontend-dev-guide/layouts/xml-manage.html" target="_blank">Common layout customization tasks</a>
-*	<a href="{{ page.baseurl }}/frontend-dev-guide/layouts/layout-extend.html" target="_blank">Extend a layout</a>
-*	<a href="{{ page.baseurl }}/frontend-dev-guide/layouts/layout-override.html" target="_blank">Override a layout</a>
-*	<a href="{{ page.baseurl }}/frontend-dev-guide/layouts/layout-practice.html" target="_blank">Customizing layout - step-by-step illustration</a>
-
-
-
+* [Layout instructions]
+* [Common layout customization tasks]
+* [Extend a layout]
+* [Override a layout]
+* [Customizing layout - step-by-step illustration]
+
+<!-- Link definitions -->
+[Layout file types]: {{ page.baseurl }}/frontend-dev-guide/layouts/layout-types.html
+[extending]: {{ page.baseurl }}/frontend-dev-guide/layouts/layout-extend.html
+[overriding]: {{ page.baseurl }}/frontend-dev-guide/layouts/layout-override.html
+[inherited]: {{ page.baseurl }}/frontend-dev-guide/themes/theme-inherit.html
+[application mode]: {{ page.baseurl }}/config-guide/bootstrap/magento-modes.html
+[Layout instructions]: {{ page.baseurl }}/frontend-dev-guide/layouts/xml-instructions.html
+[Common layout customization tasks]: {{ page.baseurl }}/frontend-dev-guide/layouts/xml-manage.html
+[Extend a layout]: {{ page.baseurl }}/frontend-dev-guide/layouts/layout-extend.html
+[Override a layout]: {{ page.baseurl }}/frontend-dev-guide/layouts/layout-override.html
```

Comment:
The url is the same as in `[overriding]` and `[override]`. Remove the duplicates and use a single reference in the topic for the link.

Reply:
fixed